AiPlatform/samples/V1/FeaturestoreServiceClient/batch_create_features.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+
+// [START aiplatform_v1_generated_FeaturestoreService_BatchCreateFeatures_sync]
+use Google\ApiCore\ApiException;
+use Google\ApiCore\OperationResponse;
+use Google\Cloud\AIPlatform\V1\BatchCreateFeaturesResponse;
+use Google\Cloud\AIPlatform\V1\CreateFeatureRequest;
+use Google\Cloud\AIPlatform\V1\Feature;
+use Google\Cloud\AIPlatform\V1\Feature\ValueType;
+use Google\Cloud\AIPlatform\V1\FeaturestoreServiceClient;
+use Google\Rpc\Status;
+
+/**
+ * Creates a batch of Features in a given EntityType.
+ *
+ * @param string $formattedParent          The resource name of the EntityType to create the batch of Features under.
+ *                                         Format:
+ *                                         `projects/{project}/locations/{location}/featurestores/{featurestore}/entityTypes/{entity_type}`
+ *                                         Please see {@see FeaturestoreServiceClient::entityTypeName()} for help formatting this field.
+ * @param string $formattedParent          The resource name of the EntityType to create a Feature.
+ *                                         Format:
+ *                                         `projects/{project}/locations/{location}/featurestores/{featurestore}/entityTypes/{entity_type}`
+ *                                         Please see {@see FeaturestoreServiceClient::entityTypeName()} for help formatting this field.
+ * @param int    $requestsFeatureValueType Immutable. Type of Feature value.
+ * @param string $requestsFeatureId        The ID to use for the Feature, which will become the final component of
+ *                                         the Feature's resource name.
+ *
+ *                                         This value may be up to 128 characters, and valid characters are
+ *                                         `[a-z0-9_]`. The first character cannot be a number.
+ *
+ *                                         The value must be unique within an EntityType.
+ */
+function batch_create_features_sample(
+    string $formattedParent,
+    string $formattedParent,
```

Comment:
This is caused by a request message being nested within another request message: https://github.com/googleapis/googleapis/blob/master/google/cloud/aiplatform/v1/featurestore_service.proto#L925-L929